### ISSUE TITLE
phpmyadmin crashes php server

### ISSUE THREAD

**Issue by oppiet30** [phpmyadmin/phpmyadmin#19358]
### Describe the bug

I am running Fedora 41, it happened in Fedora 40. I am running current version of phpMyAdmin. phpMyAdmin-5.2.1-5.fc41.noarch
I am running it on  php-8.3.13-1.fc41.rpm. Every time I open the phpMyAdmin page it gives a 503 http error. I went anduninstalled
php and then reinstalled it and the issue comes back after a little while. I don't know how to debug it, There isn't anything in /var/log/php-fpm/www-error.log. It says in /var/log/php-fpm/error.log:
[04-Nov-2024 07:59:05] WARNING: [pool www] child 3642 exited on signal 11 (SIGSEGV - core dumped) after 74323.329195 seconds from start
Every php that I run works. Just phpMyAdmin crashes the server.
I don't know how to diagnose it, except uninstall php and then reinstall it with minimum I need to run phpMyAdmin.

### Screenshots

If applicable, add screenshots to help explain the bug.

### Server configuration

- Operating system:
- Web server:
- Database version:
- PHP version:
- phpMyAdmin version:

### Client configuration

- Browser:
- Operating system:

### Additional context

Add any other context about the bug here.

**Comment by kamil-tekiela** [comment]
Can you show us your `phpinfo()`? What extensions do you have installed? Do you have anything like xDebug or uopz?

**Comment by oppiet30** [comment]
I have both xdebug and uopz.

**Comment by oppiet30** [comment]
https://linuxassistant.com/phpinfo.php

**Comment by kamil-tekiela** [comment]
> I have both xdebug and uopz.

Yeah, I am pretty sure that this is the problem. While you get a crash with phpMyAdmin, the symptoms are evident of an issue with one of your PHP extensions and these extensions are known to cause such issues. Disable them and try it again.

**Comment by oppiet30** [comment]
Yes, that worked.

**Comment by oppiet30** [comment]
It must be uopz that is causing the problem.
I put xdebug3 in and it works.

**Comment by oppiet30** [comment]
Thanks. i probably could have figured it out by uninstalling php packages until I found it.
